.github/ISSUE_TEMPLATE/Feature_requrest.md

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
@tgulacsi note the file name merged has a typo.  It should be Feature_request.md

Reply:
> @tgulacsi note the file name merged has a typo. It should be Feature_request.md

oh, damn!
I'm inattentive :(
you're right